tide-whisperer.go

```diff
@@ -296,6 +317,11 @@ func main() {
 				results = results["latest_doc"].(map[string]interface{})
 			}
 			if len(results) > 0 {
+				if results["type"].(string) == "pumpSettings" && parametersHistory != nil {
+					payload := results["payload"].(map[string]interface{})
```

Comment:
I don't understand why we want/need to insert the whole history into each entry of type "pumpSettings". This duplicates the history...